wiki/markdown/status.md

```diff
@@ -0,0 +1,33 @@
+# Status
+
+|Method|Endpoint         |Authentication|
+|------|-----------------|--------------|
+|GET   |/status          |no            |
+
+## Check the Eiffel Intelligence status
+
+An endpoint to check if Eiffel Intelligence back-end is working properly.
+The response data contains status of Eiffel Intelligence and dependent service
+such as RabbitMQ and MongoDB.
+
+There are 3 differnt kind of status indications:
+ **NOT_SET**, **AVAILABLE**, **UNAVAILABLE**
+
```

Comment:
Could you describe the definition of each status/state? What does those means? What does available, unavailable, not_set mean? What is working and not working in different states?

Reply:
Done.